### PR TITLE
vmanomaly new config 1.9.2

### PR DESCRIPTION
**Pull request by Cambaza**

### CODE REVIEW

**Comment by Cambaza** [comment]
@zekker6 can you, please, take a look?

File: charts/victoria-metrics-anomaly/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
 - TODO
 
+## 1.0.0
+
+**Release date:** 2024-02-01
+
+![AppVersion: v1.9.2](https://img.shields.io/static/v1?label=AppVersion&message=v1.9.2&color=success&logo=)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+- Add ability to render vmanomaly config as is via `vmanomalyConfig`.
```
Comment (Haleygo) [inline charts/victoria-metrics-anomaly/CHANGELOG.md:14]: Hey, change notes should be under `Next release` and new version title will be generated automatically.
As the [release CI](https://github.com/VictoriaMetrics/helm-charts/blob/master/.github/workflows/release.yaml#L56) reads content under `Next release` and determine whether to release new version.
```
## Next release

- Add ability to render vmanomaly config as is via `vmanomalyConfig`.
- xxx
```

File: charts/victoria-metrics-anomaly/CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 ## Next release
 
-- TODO
+- Add ability to render full vmanomaly config via `config` parameter.
```
Comment (zekker6) [inline charts/victoria-metrics-anomaly/CHANGELOG.md:5]: It is better to update this item to clearly state that this is a breaking change and `config` updates will be required for upgrade.
Currently, it seems like this only adds an additional field/option to provide full configuration without breaking compatibility.

**Review by zekker6 — APPROVED** [review]

**Review by dima-vm — APPROVED** [review]
Great, thanks!

**Pull request merged**